src/data/boss.json

```diff
@@ -163,6 +163,98 @@
         "wikiLink": "https://escapefromtarkov.fandom.com/wiki/knight",
         "behavior": "Rush"
     },
+    {
+        "name": "Big Pipe",
+        "normalizedName": "big-pipe",
+        "details": "The leader of 'The Goons'. Can spawn on many different maps.",
+        "health": 910,
```

Comment:
We pull Mob health from SPT. Doesn't hurt to have it here, though.

Reply:
is there a better way to pull this info? i just used the wiki site to gain the info for health